app.gummi.gummi.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018-2020 Flathub maintainers -->
+<component type="desktop">
+  <id>app.gummi.gummi</id>
+  <launchable type="desktop-id">app.gummi.gummi.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>Gummi</name>
+  <summary>The simple LaTeX editor</summary>
+  <description>
+    <p>Gummi is a LaTeX editor written in the C programming language using the GTK+ interface toolkit. It was designed with simplicity and the novice user in mind, but also offers features that speak to the more advanced user.</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Gummi Team.</p>
```

Comment:
Again, there's absolutely no reason to have that disclaimer here.

Reply:
Done https://github.com/flathub/flathub/pull/3832/commits/82bd438f779ec65dd423929a5cd745d2ead83b8e